### PR TITLE
remove(pino): no longer needed

### PR DESCRIPTION
**Pull request by peterblazejewicz**
As of version 7.0.0:
https://github.com/pinojs/pino/releases/tag/v7.0.0

Thanks!

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Delete the package's directory.
- [x] Add it to `notNeededPackages.json`.

### CODE REVIEW

File: types/ari-client/paackage.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (yorch) [inline types/ari-client/paackage.json:1]: typo in filename

Comment (yorch) [reply types/ari-client/paackage.json:1]: This typo `paackage` seems to be causing the build to fail

Comment (peterblazejewicz) [reply types/ari-client/paackage.json]: oh, yes, thank you!

**Comment by peterblazejewicz** [comment]
pending on microsoft/DefinitelyTyped-tools#344

**Comment by paambaati** [comment]
@peterblazejewicz Would you mind looking at the latest failing run?

**Comment by peterblazejewicz** [comment]
re-pushed, to re-kick ci with updated deps

**Review by yorch — COMMENTED** [review]
👍

**Review by paambaati — APPROVED** [review]

**Review by andrewbranch — APPROVED** [review]

**Comment by peterblazejewicz** [comment]
Ready to merge 💘

**Pull request merged**